### PR TITLE
add Patreon

### PR DESCRIPTION
**Pull request by tleydxdy**

### CODE REVIEW

**Comment by tleydxdy** [comment]
I've emailed the asking what are not deleted and why. They do process transactions, so maybe they have some legal limits.

**Comment by tupaschoal** [comment]
Ok, let us know when you find out!

File: _data/sites.json [hunk 1]
```diff
@@ -5046,6 +5046,15 @@
...
 
+    {
+        "name": "Patreon",
+        "url": "https://support.patreon.com/hc/en-us/articles/204605915-How-do-I-disable-my-account-",
+        "difficulty": "easy",
+        "domains": [
+            "patreon.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json]: Remove trailing whitespaces

**Comment by tupaschoal** [comment]
Did you get an answer @tleydxdy?

**Comment by OfficialMuffin** [comment]
Just to let you know @tleydxdy that the travis error you have is:

```
Line: 5057 Unexpected trailing spaces found
```

Remove those whitespaces and indent with 4 spaces from margin.

**Comment by tupaschoal** [comment]
I've merged this commit with changed difficulty. If @tleydxdy gets back with an answer from Patreon, we can change it later.

**Comment by tleydxdy** [comment]
seems like patreon have seen the light https://support.patreon.com/hc/en-us/articles/204605915-Delete-my-Patreon-account I can no longer login so I can't really try this out :(

**Pull request merged**